cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -120,6 +120,8 @@ module InstructionSanity {
     )
   }
 
+  query predicate duplicateChiOperand(ChiInstruction chi) { chi.getTotal() = chi.getPartial() }
```

Comment:
Can you make this query predicate have the form of a problem query? See `useNotDominatedByDefinition` for an example. Having the function name in the output makes it much easier to track down sanity failures in real-world snapshots.